### ISSUE TITLE
Error during plugin initialization

### ISSUE THREAD

**Issue by floriangaechter** [reimertz/gatsby-source-hire-with-google#6]
Hi @reimertz,

first of all, thanks for creating this plugin – it's just what we needed…

I'm running into an issue while using the plugin. During initialization, I get the following error message:
```
Your plugins must export known APIs from their gatsby-node.js.
The following exports aren't APIs. Perhaps you made a typo or your plugin is outdated?

See https://www.gatsbyjs.org/docs/node-apis/ for the list of Gatsby Node APIs

- The plugin "gatsby-source-hire-with-google@1.2.0" is exporting a variable named "getJobs" which isn't an API.

- The plugin "gatsby-source-hire-with-google@1.2.0" is exporting a variable named "generateBaseUrl" which isn't an API.

- The plugin "gatsby-source-hire-with-google@1.2.0" is exporting a variable named "generateBaseUrl" which isn't an API.
```

I'm using Gatsby version 2.4.6.

The plugin works fine, as in it fetches the data from the Google Hire API and I can access it through GraphQL. I'm pretty new with Gatsby and I can't really pinpoint what's the issue but I assume Gatsby changed how their gatsby-node.js APIs work?

**Comment by reimertz** [comment]
Hi @floriangaechter and thanks! I created it for a project that ended up not using it so glad someone else find it useful.  :)

Regarding the error, it definitely seems like I'm not hooking everything up properly anymore. Let me investigate.

**Comment by reimertz** [comment]
Hey @floriangaechter, I just published v1.3.0  that should contain fixes for your error messages. The issue was that I exported some helper functions so I could run my tests but Gatsby didn't like that so had to do some refactoring. 

All should work now. :D Please confirm, and if it's the case, please close the issue. <3

**Comment by floriangaechter** [comment]
Hi @reimertz, sweet! It works perfectly! Thanks so much for taking a look and resolving the issue so fast ❤
